src/components/MainContainer/MainContainer.jsx

```diff
@@ -2,17 +2,41 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './MainContainer.module.css';
 import FilmsContainer from '../FilmsContainer/';
+import { connect } from 'react-redux';
+import NotFound from '../NotFound';
+import { loadData } from './connect-store';
 
-function MainContainer(props) {
-    return (
-        <main className={styles.mainContainer}>
-            {props.films ? <FilmsContainer films={props.films} /> : null}
-        </main>
-    );
+class MainContainer extends React.Component {
+    componentDidMount() {
+        this.props.loadData();
+    }
+
+    render() {
+        let content = <NotFound />;
+        if (this.props.films) {
+            if (this.props.films.length > 0) {
+                content = <FilmsContainer films={this.props.films} />;
+            }
+        }
+
+        return <main className={styles.mainContainer}>{content}</main>;
```

Comment:
you can write like this 
`<main className={styles.mainContainer}>{(this.props.films && this.props.films.length) ?<FilmsContainer films={this.props.films} /> : <NotFound /> }`

Reply:
Fixed